helm/openwhisk/templates/_invoker-helpers.tpl

```diff
@@ -33,8 +33,12 @@
 
 {{- define "openwhisk.docker_pull_runtimes" -}}
 - name: docker-pull-runtimes
-  imagePullPolicy: {{ .Values.utility.imagePullPolicy | quote }}
+{{- if .Values.docker.usePrivateRegistry }}
```

Comment:
Why put this in an if/else block instead of letting .Values.docker.registry.name default to an empty string?

Reply:
Hi @dgrove-oss, I used this based on the usePrivateRegistry flag keeping a scenario where the user won't like to use private Docker registry for component images but want to use only for runtime images (like today's behavior).

Though using your method is better, in that case, one has to put all component + runtime images in Private Docker Registry is registry name is provided. Let me know your thoughts, I will modify it based on your suggestion.

Reply:
Hi @dgrove-oss, I modified it based on your suggestion and removed an extra flag(usePrivateRegistry). Now if docker registry is not empty, then charts will use imagePullSecrets and docker registry provided to pull the images. Let me know if you have further suggestions.

Reply:
This looks reasonable to me.  Is it still a WIP or is it ready for merging?

Reply:
Thanks @dgrove-oss, it is ready to be merged now.